packages/manager/modules/manager-components/src/edit-name/template.html

```diff
@@ -0,0 +1,29 @@
+<form
+    id="serverEditNameForm"
+    name="serverEditNameForm"
+    data-ng-submit="serverEditNameForm.$valid && $ctrl.saveName()"
+    novalidate
+>
+    <oui-modal
+        data-heading="{{:: 'server_name_edit_title' | translate }}"
+        data-primary-label="{{:: 'server_name_edit_common_save' | translate }}"
+        data-secondary-label="{{:: 'server_name_edit_common_cancel' | translate }}"
+        data-secondary-action="$ctrl.handleCancel()"
+        data-on-dismiss="$ctrl.handleCancel()"
+        data-loading="$ctrl.loading || $ctrl.isUpdating"
```

Comment:
i am not able to see the ```data-primary-action``` to applied name change, normal ?

Reply:
Yes its normal. I am using form submit instead of primary action.